packages/dds/tree/src/feature-libraries/object-forest/objectForest.ts

```diff
@@ -43,6 +43,11 @@ export class ObjectForest extends SimpleDependee implements IEditableForest {
     }
 
     public root(range: DetachedField, index = 0): Anchor {
+        if (!this.roots.get(range)?.length) {
+            return this.anchors.track(null);
+        }
```

Comment:
I'm guessing the `this.anchors.track(null)` returns some kind of invalid anchor. Why don't we want to return a normal anchor here? The fact that it may point to some non-existent place shouldn't be a problem for the anchor, right?

Reply:
We wanted an anchor that points to nothing and will always point to nothing, no matter what happens to the root trait. Can we construct such an anchor without making a special one?

Reply:
After chatting with Noah, I can see how it may be easier to have this NeverAnchor concept rather than having burden the normal anchor management code with this possibility. Resolving this comment.